### PR TITLE
AshwinHPrasad simple calculator 

### PR DESCRIPTION
**Pull request by ashwinhprasad**
simple calculator of c++ is completed

### CODE REVIEW

File: calculator.cpp [hunk 1]
```diff
@@ -2,26 +2,33 @@
...
 int division(int number_1, int number_2){
-
+    return number_1/number_2;
 }
 
 int get_input(){
-
+    int a,b;
+    cout<<"Enter 2 numbers"<<endl;
+    cin>>a>>b;
```
Comment (surabhiojha12) [inline calculator.cpp]: use proper variable name

File: calculator.cpp [hunk 2]
```diff
@@ -2,26 +2,33 @@
...
 }
 
 int multiplication(int number_1, int number_2){
-
+    return number_1*number_2;
 }
 
 int division(int number_1, int number_2){
-
+    return number_1/number_2;
```
Comment (surabhiojha12) [inline calculator.cpp]: divide by 0 case?

File: calculator.cpp (hunk 1 above)
Comment (ashwinhprasad) [reply calculator.cpp]: okay mam

File: calculator.cpp (hunk 2 above)
Comment (ashwinhprasad) [reply calculator.cpp]: okay mam. added

**Pull request closed without merging**